inginious/frontend/pages/course_admin/task_edit.py

```diff
@@ -39,8 +39,14 @@ def GET_AUTH(self, courseid, taskid):  # pylint: disable=arguments-differ
             task_data = None
         if task_data is None:
             task_data = {}
-
+        
+        # Sorting to asure that multilang is always the first option
         environments = self.containers
+        environments = list(environments)
+        environments.remove("multiple_languages")
+        environments.insert(0,"multiple_languages")
+        environments = tuple(environments)
```

Comment:
Please check that "multiple_languages" is actually on the list to avoid an exception.